### PR TITLE
Add Hashtags to Share Urls

### PR DESCRIPTION
**Pull request by Syralist**
Twitter and Facebook allow the addition of hashtags to the share urls, so I added this to the "post-share" partial. It takes the posts tags and appends them to the Urls.
Twitter allows a list of tags, Facebook only allows one tag.
It works but I don't know if the code is good.

### CODE REVIEW

**Comment by reuixiy** [comment]
Yup, it's great!

File: layouts/partials/components/post-share.html [hunk 1]
```diff
@@ -1,5 +1,21 @@
 {{ if and .Site.Params.enablePostShare (.Params.share | default .Site.Params.displayPostShare) }}
 
+    {{- $hashtags := newScratch -}}
+    {{ if .Site.Params.addHashtags }}
```
Comment (reuixiy) [inline layouts/partials/components/post-share.html:4]: I think maybe we can remove this param?

**Comment by reuixiy** [comment]
@Syralist I made some adjustments and I'm going to merge this PR if there is no problem.

**Comment by reuixiy** [comment]
Okay, I tested that, seems like Hugo has some kind of "magic", which lowercase `.Param.tags`...

**Comment by Syralist** [comment]
Yes, I guess the `.Param.tags` works for the urls to the respective pages. The code I copied comes from the `post-tags` partial.

**Comment by reuixiy** [comment]
So I'm going to merge this PR now. Do you have any other suggestions?

**Comment by Syralist** [comment]
No suggestions at the moment. :)

**Pull request merged**